torchvision/transforms/v2/_auto_augment.py

```diff
@@ -15,6 +15,59 @@
 from .utils import check_type, is_simple_tensor
 
 
+def solarize_add(
+    image: Union[datapoints._ImageType, datapoints._VideoType], addition: float = 0.0, threshold: float = 128 / 255
+) -> Union[datapoints._ImageType, datapoints._VideoType]:
+    if check_type(image, (datapoints.Image, is_simple_tensor, datapoints.Video)):
+        bound = _FT._max_value(image.dtype)
+    else:
+        bound = 255
+    if bound != 1:
+        addition = round(addition * bound)
+        threshold = round(threshold * bound)
+
+    if isinstance(image, PIL.Image.Image):
+
+        def pixel_fn(pixel: int) -> int:
+            return max(0, min(pixel + int(addition), bound)) if pixel < int(threshold) else pixel
```

Comment:
`int()` convert is only for mypy type checking and unnecessary during execution.